google/api_core/gapic_v1/method.py

```diff
@@ -117,19 +135,21 @@ def wrap_method(
     func,
     default_retry=None,
     default_timeout=None,
+    default_compression=None,
     client_info=client_info.DEFAULT_CLIENT_INFO,
 ):
     """Wrap an RPC method with common behavior.
 
-    This applies common error wrapping, retry, and timeout behavior a function.
-    The wrapped function will take optional ``retry`` and ``timeout``
+    This applies common error wrapping, retry, timeout, and compression behavior a function.
```

Comment:
Nit: I think there might be a word missing here? This doesn't read like a complete sentence to me. Perhaps it should be: "This applies common error wrapping, retry, timeout, and compression behavior to a function."